manifests/server/config.pp

```diff
@@ -120,14 +120,16 @@
         }
       }
       if versioncmp($facts['os']['release']['major'], '8') >= 0 {
-        systemd::dropin_file { 'puppet.conf':
-          unit    => "${openldap::server::service}.service",
-          content => join([
-              '[Service]',
-              'EnvironmentFile=/etc/sysconfig/slapd',
-              'ExecStart=',
-              "ExecStart=/usr/sbin/slapd -u ${openldap::server::owner} -h \${SLAPD_URLS} \$SLAPD_OPTIONS",
-          ], "\n"),
+        if $openldap::server::package == 'openldap-servers' {
+          systemd::dropin_file { 'puppet.conf':
+            unit    => "${openldap::server::service}.service",
+            content => join([
```

Comment:
Switch to https://github.com/voxpupuli/puppet-systemd/blob/master/REFERENCE.md#systemd--manage_dropin ?

Reply:
I guess.  Principle of least-surprise says that's not necessary here but if you need me to polish that then I guess.